source/support/support-overview.rst

```diff
@@ -0,0 +1,179 @@
+=======
+Support
+=======
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Overview
+--------
+
+MinIO users have two options for support.
+
+#. Community support from the `public Slack channel <https://minio.slack.com>`_.
```

Comment:
Should add something like "Community support is best-effort only and has no SLA or SLO"

Could do :abbr:`SLA (Service Level Agreement)` or :abbr:`SLO (Service Level Objective)` here. Not sure if I have the syntax right.